templates/waggledance.json

```diff
@@ -50,7 +55,11 @@
 	  {
 		"name": "LOG4J_FORMAT_MSG_NO_LOOKUPS",
 		"value": "true"
-	  }
+	  },
+	{
```

Comment:
formatting needed. Align it like the other blocks above.

Reply:
done

Reply:
I don't see it fixed - https://github.com/ExpediaGroup/apiary-federation/blob/76f8ab5919b1ce7a4e006bba3c7f7cd005c79546/templates/waggledance.json